specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/managedInstances.json

```diff
@@ -548,22 +548,9 @@
             "modelAsString": true
           },
           "x-ms-mutability": [
+            "read",
             "create"
           ]
-        },
-        "currentBackupStorageRedundancy": {
```

Comment:
This is a breaking change, that would involve a breaking change in any SDKs generated from this swagger previously.  Do you have any non-preview SDK packages released with this version?